getting_started/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -194,3 +207,36 @@ Constants
 
 Use CONSTANT\_CASE, all caps, with an underscore (\_) to separate words:
 ``const MAX_SPEED = 200``
+
+Static Typing
+-------------
+
+Since Godot 3.1, GDScript supports :ref:`optional static typing<_doc_gdscript_static_typing>`.
```

Comment:
The leading `_` should not be used in the hyperlink, so it should be `:ref:`optional static typing <doc_gdscript_static_typing>`.